scanner/redhatbase.go

```diff
@@ -85,6 +89,27 @@ func detectRedhat(c config.ServerInfo) (bool, osTypeInterface) {
 		}
 	}
 
+	if r := exec(c, "ls /etc/almalinux-release", noSudo); r.isSuccess() {
+		if r := exec(c, "cat /etc/alma-release", noSudo); r.isSuccess() {
```

Comment:
Isn't `cat /etc/alma-release` a typo for `cat /etc/almalinux-release`?

Reply:
thankyou! 
I fixed.